modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -230,7 +230,7 @@ The following elements were previously deprecated and have, with this release, b
 
 | `images_dataimg_filter`               | Option        |
 
-| `JSON`                                | API           |
+| JSON                                  | API           | Use the native JSON https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON[API] instead.
```

Comment:
How come we removed the backticks here? That doesn't look right and is inconsistent.

Reply:
Because I’m an idiot and over-relied on copy-and-paste from my own Jira ticket. <sigh>

Corrected on both lines.